easy-ui-react/src/utilities/css.ts

```diff
@@ -76,7 +76,7 @@ export function getResponsiveDesignToken(
     return getComponentToken(
       componentName,
       `${componentProp}-xs`,
-      `var(--ezui-${kebabCase(tokenSubgroup)}-${responsiveProp})`,
+      `var(--ezui-${kebabCase(tokenSubgroup)}-${kebabCase(responsiveProp)})`,
```

Comment:
**Question:**
Was this a known bug or had we just not used `responsiveProp` yet?

Reply:
we just didn't use a `responsiveProp` that needed to be kebab'd yet. i think i ended up removing what i used it for in a Story for this card, but i kept the fix in. reminds me i need to add a test for it